sklearn/preprocessing/_data.py

```diff
@@ -415,6 +422,8 @@ def fit(self, X, y=None):
         self : object
             Fitted scaler.
         """
+        self._validate_params()
+
```

Comment:
Since `fit` is directly calling `partial_fit` which does param validation, there's no need to validate here as well.
```suggestion
```